pkg/web/buildsHandler.go

```diff
@@ -25,6 +32,9 @@ func (mbh *MobileBuildsHandler) List(c echo.Context) error {
 		c.Logger().Errorf("error listing builds %v", err)
 		return c.NoContent(http.StatusInternalServerError)
 	}
-
-	return c.JSON(http.StatusOK, builds)
+	response := &response{
+		BuildList:          *builds,
+		OpenshiftMasterURL: mbh.openshiftMasterURL,
```

Comment:
rather than return the url on its own, I think maybe use it to construct the url of each build, and then add the url to each build as a separate field (like buildURL)? There are at least 2 benefits:

1. Cleaner API. Otherwise without some documentation, people wouldn't be able to understand why the master url is returned here.
2. Hide the logic to build the full url of the builds. This only needs to be done on the server and the clients don't need to care.

WDYT?

Reply:
I think it sounds reasonable. This will definitely reduce amount of changes needed in UI code to reflect this, which I'm looking at right now. Thanks